### ISSUE TITLE
GPRS Server security

### ISSUE THREAD

**Issue by mohanbabi** [KenLagoni/MavlinkGPRS#5]
Just idea for increasing security, user friendly.
![GPRS Server security](https://user-images.githubusercontent.com/22990144/129664089-8f6ad588-e4d7-4f06-b997-c4be17d42b81.png)

Sir if any mistake sorry😀

**Comment by KenLagoni** [comment]
The problem as I see it, is that Qgroundcontrol or mission planner doesn't support login/password.

**Comment by mohanbabi** [comment]
OK @KenLagoni  Sir,

**Comment by mohanbabi** [comment]
Hi sir how are you @KenLagoni

**Comment by KenLagoni** [comment]
> Hi sir how are you @KenLagoni

Fine Thank you, just busy with work :-) - Have you had any luck flying with the GPRS?

**Comment by mohanbabi** [comment]
No, @KenLagoni Sir. I am also busy with medicine delivery trials. this week my team is on the medicine delivery mission.
After this work, I will plan to test your GPRS link test. I am planning to make a home mavlink server, using raspberry pi.

I Will give updates on my testing sir
Thank you @KenLagoni Sir Take care

**Comment by mohanbabi** [comment]
@KenLagoni  Sir you have any youtube channels, if yes please give your profile link. please

**Comment by KenLagoni** [comment]
> @KenLagoni Sir you have any youtube channels, if yes please give your profile link. please

HI, no not really, only one video :-) 
https://www.youtube.com/watch?v=SjNEsUAFRt4&t=86s

I'm working on the LTE project, I hope to finish first release and documentation this weekend.

**Comment by mohanbabi** [comment]
@KenLagoni  that is great news sir...i am also planning for testing of your's 2g this weekend. we have YouTube channel.... i will give credit for you sir....
https://youtube.com/channel/UCJk32MAm30fJilUG4Ad8dzg

i am camera man😀

**Comment by mohanbabi** [comment]
![image](https://user-images.githubusercontent.com/22990144/137590122-a1a7ea68-cbf8-4446-9872-5513e9dcb3b8.png)


port forward is not working sir any suggestion

**Comment by mohanbabi** [comment]
![image](https://user-images.githubusercontent.com/22990144/137590174-cc7b5469-cd92-4464-8192-6f2781184d59.png)

**Comment by KenLagoni** [comment]
So the MavlinkGPRS will not reply to canyouseeme.org. If I do that mine says error to.

Have you tried and see what the tcpdump outputs when you do that test? - If I go to canyouseeme.org and try port 14450 when tcpdump is listing on that port, I get an error to, but the tcpdump actually sees the request:
![image](https://user-images.githubusercontent.com/43088513/137623579-7c93797d-6346-4482-ba49-e464363a91b8.png)

**Comment by KenLagoni** [comment]
> @KenLagoni that is great news sir...i am also planning for testing of your's 2g this weekend. we have YouTube channel.... i will give credit for you sir.... https://youtube.com/channel/UCJk32MAm30fJilUG4Ad8dzg
> 
> i am camera man😀

Nice channel I will check it out :-)

**Comment by mohanbabi** [comment]
![Screenshot from 2021-10-17 16-47-07](https://user-images.githubusercontent.com/22990144/137625117-927f03de-03b9-41b8-8baf-d5afc7cc2781.png)
sir my response can I go with testing. I am struggling with 2 days.  
first-time canyouseenmyip.org is worked, but this time it is not worked.

**Comment by mohanbabi** [comment]
due to this problem only I canceled flying 

thank you for this command "sudo tcpdump port 14450". there is one person online explain about command.

@KenLagoni  Sir, now can we plan for flying

**Comment by KenLagoni** [comment]
> ![Screenshot from 2021-10-17 16-47-07](https://user-images.githubusercontent.com/22990144/137625117-927f03de-03b9-41b8-8baf-d5afc7cc2781.png) sir my response can I go with testing. I am struggling with 2 days. first-time canyouseenmyip.org is worked, but this time it is not worked.

Looks like your port forward is working.

**Comment by mohanbabi** [comment]
@KenLagoni  thank you Sir, can I go for fly sir
